test/functional/tests.py

```diff
@@ -807,6 +807,68 @@ def test_bitswap_unwant(self):
         result = self.api.bitswap_unwant(key='QmZTR5bcpQD7cFgTorqxZDYaew1Wqgfbd2ud9QqGPAkK2V')
         self.assertTrue(result is not None)
 
+@skipIfOffline()
+class IpfsApiPubSubTest(unittest.TestCase):
+
+    def setUp(self):
+        self.api = ipfsapi.Client()
+
+    def test_pubsub_pub_sub(self):
+        """
+        We should test both publishing and subscribing
+        at the same time for it's difficult
+        to determine if one works without checking the
+        other.
+        """
```

Comment:
This sentence confused me originally when reading it for the first time: At least to me it sounds like a TODO-entry. Is this supposed to be present-tense (ie: “We test both publishing and subscribing at the same time because …”) by any chance?